dashboard/config/blocks/GamelabJr/gamelab_hideVariable.json

```diff
@@ -17,4 +12,4 @@
     "inline": true,
     "style": "variable_blocks"
   }
-}
\ No newline at end of file
+}
```

Comment:
Nit: This line should change back if you run `bundle exec rake seed:blocks` from /dashboard. You can also make these changes from the levelbuilder interface if you want to avoid this. I don't think it matters so much, but if we leave it like this, it will probably show up as an extra diff back to the original end-of-file somewhere in our deploy pipeline.

Reply:
Here's a commit from my recent branch where I undid some of VSCode's changes by reseeding. You can see my editor also changed some other whitespace stuff, which doesn't apply to your new blocks.  https://github.com/code-dot-org/code-dot-org/commit/2a3947f53ccb948935eb47af3385a14617622e12